content/ops/contingency-plan.md

```diff
@@ -70,10 +70,12 @@ If GitHub becomes unavailable, the live cloud.gov will continue to operate in it
 #### Disruption lasting less than 7 days
 Cloud Operations will postpone any non-critical updates to the platform until the disruption is resolved.  If a critical update **must** be deployed, Cloud Operations will:
 
-* Locate a copy of the current version of the required [git](https://git-scm.com/) repository by comparing last commit times of all checked out versions on Cloud Operations local systems and any copies held by cloud.gov systems (Concourse, BOSH director, etc.) Cloud Operations can also coordinate with the 18F Infrastructure team to retrieve 18F's org-wide GitHub backups, which are stored in [Backhub](https://backhub.co/).
+* Locate a copy of the current version of the required [git](https://git-scm.com/) repository by comparing last commit times of all checked out versions on Cloud Operations local systems and any copies held by cloud.gov systems (Concourse, BOSH director, etc.) 
 * Pair with another member of Cloud Operations to:
-  * Perform the change on the local copy of the repository
-  * Manually deploy the change by provisioning a Concourse jumpbox container, copying in the repository, and executing any required steps by hand
+  * Perform the change on the local copy of the repository (if the update requires a change to git-managed source code, or
```

Comment:
For readability, I'd suggest moving the second bullet point into this line, since this is basically one step with two forks.